src/containers/shared/components/Transaction/AMMBid/Simple.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleProps } from '../types'
+import { localizeBalance } from '../../../utils'
+import { Account } from '../../Account'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { ammAccountID, bidMin, bidMax, authAccounts } = data.instructions
+  const localizedBidMin = localizeBalance(bidMin)
+  const localizedBidMax = localizeBalance(bidMax)
+
+  return (
+    <>
+      {ammAccountID && (
+        <SimpleRow label={t('amm_account_id')} data-test="account_id">
+          <Account account={ammAccountID} />
+        </SimpleRow>
+      )}
+      {localizedBidMin && (
+        <SimpleRow label={t('min_slot_price')} data-test="min_slot_price">
+          {localizedBidMin}
+        </SimpleRow>
+      )}
+      {localizedBidMax && (
+        <SimpleRow label={t('max_slot_price')} data-test="max_slot_price">
+          {localizedBidMax}
+        </SimpleRow>
+      )}
+      {authAccounts && (
```

Comment:
Do you have an example screenshot for how the Auth Accounts look on the Simple page? I want to make sure it shows up correctly

Reply:
There are unit tests for this reason - and i dont currently have an example but I have tested it manually as well. Feel free to replicate it yourself. You can use this script to make the accounts:

https://github.com/gregtatcam/xrpl-utils/blob/main/amm/amm_tester.py